test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/EverythingOperationTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Net;
+using Microsoft.Health.Fhir.Client;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
+using Microsoft.Health.Fhir.Tests.E2E.Rest.Search;
+using Microsoft.Health.Test.Utilities;
+using Xunit;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.Health.Fhir.Tests.E2E.Rest
+{
+    [HttpIntegrationFixtureArgumentSets(DataStore.All, Format.Json)]
+    public class EverythingOperationTests : SearchTestsBase<EverythingOperationTestFixture>
+    {
+        public EverythingOperationTests(EverythingOperationTestFixture fixture)
```

Comment:
>EverythingOperationTests [](http://example.com/codeflow?start=15&length=24)

Is any of these tests triggers continuation token and iterate with it over collection?

Reply:
Yes, tests that use `ExecuteAndValidateBundle` trigger continuation token to get and validate the second page as well. I also refined the first test to iterate over all resources in response across three pages.